tableau-databricks/connection-builder.js

```diff
@@ -38,13 +38,6 @@ limitations under the License.
 
 	var authenticationMode = attr[connectionHelper.attributeAuthentication];
 	switch (authenticationMode) {
-		case "auth-user-pass":
```

Comment:
Oh actually let's keep this file unchanged, there may be existing tableau connection which use this and we wanna them to see correct error message thrown by Databricks.

Reply:
Done